lib/faerie/src/backend.rs

```diff
@@ -238,7 +258,11 @@ impl Backend for FaerieBackend {
 /// [`finish`](../cretonne_module/struct.Module.html#method.finish) function.
 /// It provides functions for writing out the object file to memory or a file.
 pub struct FaerieProduct {
-    artifact: faerie::Artifact,
+    /// Faerie artifact with all functions, data, and links from the module defined
+    pub artifact: faerie::Artifact,
+    /// Optional trap manifest
+    pub trap_manifest: Option<FaerieTrapManifest>,
```

Comment:
Could you add a comment mentioning how to enable this (using the `FaerieBuilder`)?